en/developer_guide/routing.rst

```diff
@@ -69,28 +69,40 @@ it will be redirect to the ``http://www.open-orchestra.com`` website.
 If a user come on ``http://example.fr/fr/open-orchestra`` so the french site with the good pattern,
 it will be redirect to the ``HOME`` node of the french page.
 
-But if a user come on ``http://example.fr/fr/oo``, it will have a 404 page not found error because redirections are attached to languages.
+But if a user come on ``http://example.fr/fr/oo``, it will have a 404 page not found error because redirections
+are attached to languages.
 
-Implementation
---------------
+Implementations
+---------------
 
-``open-orchestra-front-bundle`` contains all the routing related classes in the ``routing`` folder.
+In Open Orchestra, we provide two alternative to do the routing :
 
+* Get the url directly from the database (default configuration)
+* Dump the route in a cache file
```

Comment:
You should quickly present the pro/cons of each mechanism to let the user understand why there is two mechannisms and give it arguments to choose one or the other.